iceoryx_utils/include/iceoryx_utils/posix_wrapper/thread.hpp

```diff
@@ -23,7 +23,7 @@ namespace iox
 {
 namespace posix
 {
-constexpr uint8_t MAX_THREAD_NAME_LENGTH = 15U;
+constexpr size_t MAX_THREAD_NAME_LENGTH = 15U;
```

Comment:
according to  @elfenpiff `sise_t` made problems with the ports to mac and windows and he suggested to just use `uint64_t`